config.fullstack.test.yaml

```diff
@@ -0,0 +1,248 @@
+                      x-modules:
+                        - path: sys/page_revisions.js
+                    /table: &table_spec
+                      x-modules:
+                        - path: sys/table.js
+                          options:
+                            conf:
+                              backend: '{env(RB_TEST_BACKEND, sqlite)}'
+                              hosts: [localhost]
+                              keyspace: system
+                              username: cassandra
+                              password: cassandra
+                              defaultConsistency: one # or 'localQuorum' for production
+                              storage_groups:
+                                - name: test.group.local
+                                  domains: /./
+                              dbname: test.db.sqlite3 # ignored in cassandra, but useful in SQLite testing
+
+wikimedia_project: &wikimedia_project
+  x-modules:
+    - spec:
+        paths:
+          /{api:v1}:
+            x-modules:
+              - path: projects/v1/wikimedia.wmf.yaml
+                options:
+                  <<: *default_options
+                  pageviews:
+                    host: https://wikimedia.org/api/rest_v1/metrics
+              - path: projects/proxy.yaml
+                options:
+                  backend_host_template: '{{"/{domain}/sys/legacy"}}'
+                  block_external_reqs: true
+          /{api:sys}:
+            x-modules:
+              - path: projects/proxy.yaml
+                options:
+                  backend_host_template: '{{"/{domain}/sys/legacy"}}'
+              - spec:
+                  paths:
+                    /legacy/key_value:
+                      x-modules:
+                        - path: sys/key_value.js
+                    /table: *table_spec
+                    /mathoid:
+                      x-modules:
+                        - path: sys/mathoid.js
+                          options: *mathoid_options
+                    /post_data:
+                      x-modules:
+                        - path: sys/post_data.js
+                    /events:
+                      x-modules:
+                        - path: sys/events.js
+
+wikipedia_project: &wikipedia_project
+  x-modules:
+    - spec:
+        paths:
+          /{api:v1}:
+            x-modules:
+              - path: projects/v1/wikipedia.wmf.yaml
+                options: *default_options
+              - path: test/test_module.yaml
+                options:
+                  events:
+                    uri: http://127.0.0.1:8085/v1/events
+                    topic: resource_change
+                    transcludes_topic: change-prop.transcludes.resource-change
+              - path: projects/proxy.yaml
+                options:
+                  backend_host_template: '{{"/{domain}/sys/legacy"}}'
+                  block_external_reqs: true
+          /{api:sys}: *default_sys
+
+enwiki_project: &enwiki_project
```

Comment:
Idem as for front-end, let's remove this and use only `wikipedia_project`

Reply:
No, can't do it, we add test_module to one of them and not another. For a reason of one is used to validate normal spec, one is used to do tests that use test_module